packages/react-storybook-addon/src/decorators/withFluentProvider.tsx

```diff
@@ -14,7 +13,7 @@ const getActiveFluentTheme = (globals: FluentGlobals) => {
   return { theme };
 };
 
-export const withFluentProvider = (StoryFn: StoryFunction<React.ReactElement>, context: FluentStoryContext) => {
+export const withFluentProvider = (StoryFn: () => JSX.Element, context: FluentStoryContext) => {
```

Comment:
`StoryFunction<React.ReactElement>` must have changed in 6.4, as it now requires two params. Can tighten this up later if necessary.